frontend/packages/core/src/AppLayout/notifications.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import styled from "@emotion/styled";
+import {
+  Box,
+  ClickAwayListener,
+  Grow,
+  IconButton,
+  MenuList,
+  Paper,
+  Popper,
+} from "@material-ui/core";
+import NotificationsIcon from "@material-ui/icons/Notifications";
+
+const StyledNotificationsIcon = styled(IconButton)`
+  color: #ffffff;
+  margin-right: 1rem;
+  border-radius: 8px;
+  padding: 8px;
+  &:hover {
+    background: #2d3db4;
+  }
+  &:active {
+    background: #2938a5;
+  }
+`;
+
+const Notifications: React.FC = () => {
+  const [open, setOpen] = React.useState(false);
+  const anchorRef = React.useRef(null);
+
+  const handleToggle = () => {
+    setOpen(!open);
+  };
+
+  const handleClose = event => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
+      return;
+    }
+    setOpen(false);
+  };
+
+  function handleListKeyDown(event) {
+    if (event.key === "Tab") {
+      event.preventDefault();
+      setOpen(false);
+    }
+  }
+
+  return (
+    <Box>
+      <StyledNotificationsIcon
+        ref={anchorRef}
+        edge="end"
+        aria-controls={open ? "notifcation-options" : undefined}
+        aria-haspopup="true"
+        onClick={handleToggle}
+      >
+        <NotificationsIcon />
+      </StyledNotificationsIcon>
+      <Popper open={open} anchorEl={anchorRef.current} role={undefined} transition>
+        {({ TransitionProps, placement }) => (
+          <Grow
+            {...TransitionProps}
+            style={{ transformOrigin: placement === "bottom" ? "center top" : "center bottom" }}
+          >
+            <Paper>
+              <ClickAwayListener onClickAway={handleClose}>
+                <MenuList
+                  autoFocusItem={open}
+                  id="notifcation-options"
```

Comment:
```suggestion
```

Reply:
I added it b/c I saw it included [here](https://github.com/lyft/clutch/blob/f9eb24f37b203c67037fa4071020e5511efa2563/frontend/packages/core/src/AppLayout/user.tsx#L100) and in the material ui examples. I removed it locally tho and it didn't break anything but also unsure if it's b/c of an edge case that I didn't hit. From quick research, seems like it's a common property included to create the relationship b/w a component and an expanded panel

Reply:
ah yeah you are right. Sorry about that. It does create a relationship between the modal and the button